src/services/blogApi.js

```diff
@@ -0,0 +1,29 @@
+export const getUsers = () => {
```

Comment:
```suggestion
const ROOT_ENDPOINT = 'https://jsonplaceholder.typicode.com';
const USERS_URL = `${ROOT_ENDPOINT}/users`;
const POSTS_URL = `${ROOT_ENDPOINT}/posts`;
const COMMENTS_URL = `${ROOT_ENDPOINT}/comments`;

export const getUsers = () => {
  return fetch(USERS_URL)
    .then(res => res.json());
};

export const getUser = id => {
  return fetch(`${USERS_URL}/${id}`)
    .then(res => res.json());
};

export const getPosts = () => {
  return fetch(POSTS_URL)
    .then(res => res.json());
};

export const getPost = id => {
  return fetch(`${POSTS_URL}/${id}`)
    .then(res => res.json());
};

export const getComments = id => {
  return fetch(`${COMMENTS_URL}?postId=${id}`)
    .then(res => res.json());
};

export const getPostsByUserId = id => {
  return fetch(`${POSTS_URL}?userId=${id}`)
    .then(res => res.json());
};
```
Another way to setup your api file.